dockerdb/constants.py

```diff
@@ -0,0 +1,14 @@
+from os import getenv
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+PG_DATABASE = getenv(key="PG_DATABASE", default="piccolo_database")
+PG_HOST = "localhost"
+PG_IMAGE_NAME = getenv(key="POSTGRES_IMAGE_NAME", default="postgres:latest")
+PG_PASSWORD = getenv(key="PG_PASSWORD")
+PG_PORT = getenv(key="PG_PORT", default="5432")
+PG_USER = "postgres"
+UNIQUE_CONTAINER_NAME = "piccolo_postgres_7677f8bd"
+UNIQUE_CONTAINER_LABELS = {"f2c62b9d": "2eb8705fda65"}
```

Comment:
I think it's nice to get the database details from environment variables, or a `.env` file.

I originally thought about getting the database details from `piccolo_conf.py`, by reading them from the engine. But I think this makes sense. You could have a single .env file, and `piccolo_conf.py` could read the values from there, and likewise so can this tool.